pkg/integration/utils_test.go

```diff
@@ -14,30 +14,38 @@ import (
 )
 
 func TestIsKilledFalseWithNonKilledProcess(t *testing.T) {
-	// TODO Windows: Port this test
-	if runtime.GOOS == "windows" {
-		t.Skip("Needs porting to Windows")
+	var lsCmd *exec.Cmd
+	if runtime.GOOS != "windows" {
+		lsCmd = exec.Command("ls")
+	} else {
+		lsCmd = exec.Command("cmd", "/c", "dir")
+	}
+
+	err := lsCmd.Start()
```

Comment:
Confused by the usage here.
Why not `if err := lsCmd.Run(); err != nil { t.Fatal(err) }`